demo

```diff
@@ -54,6 +69,14 @@ while [ -n "$1" ]; do
     shift
 done
 
+if [ $CLIENT = 1 ]; then
```

Comment:
Makes sense to change to arithmetic comparison, but I have never seen that syntax before. This is arithmetic comparison to me:

```
if [ $CLIENT -eq 1 ]; then
```

This is used in a lot of places, so I would favor that instead of something else which does the same thing.

Reply:
Just as good :)